packages/core-mobile/app/services/send/types.ts

```diff
@@ -51,6 +53,11 @@ export type GetTransactionRequestParams = SendServiceFuncParams
 
 export type ValidateStateAndCalculateFeesParams = SendServiceFuncParams & {
   nativeTokenBalance?: bigint // in wei
+
+  // TODO:
+  // https://ava-labs.atlassian.net/browse/CP-9142
+  // refactor ValidateStateAndCalculateFees logic in SendService
+  utxos?: BitcoinInputUTXO[] // temp field to make validating btc faster by not re-fetching balance every time
```

Comment:
this is a temp solution until we refactor the fee + validation code to be cleaner and more efficient